lib/identity/client.js

```diff
@@ -209,6 +209,41 @@ class Client extends PartialClient {
     }
     return { exists: false, results: null }
   }
+  /**
+   * getSharedSecrets
+   *
+   *
+   * @return {Promise<Record>}
+   */
+  async getSharedSecrets() {
```

Comment:
If this isn't needed for the current set of functionality that is deployed or is about to be deployed, probably best to remove this from this pr, otherwise if this is needing to be merged please fill out the documentation for this method